ci-templates/gitlab/diagram-cache.yml

```
@@ -9,30 +9,24 @@ update_capella_diagram_cache:
     name: $DOCKER_REGISTRY/capella/base:${CAPELLA_DOCKER_IMAGES_TAG}
     entrypoint: [""]
   script:
-    - pip install capellambse
+    - pip install "capellambse[cli]@git+https://github.com/DSD-DBS/py-capellambse.git@${CAPELLAMBSE_REVISION}"
     - mkdir diagram_cache
+    - export DISPLAY=:99
+    - Xvfb :99 -screen 0 1920x1080x8 -nolisten tcp &
+    - python -m capellambse.diagram_cache
```

```suggestion
    - xvfb-run python -m capellambse.diagram_cache
```

I'd prefer [`xvfb-run`](https://manpages.ubuntu.com/manpages/xenial/man1/xvfb-run.1.html ) over `Xvfb` because xvfb-run automatically terminates the virtual display when it's not needed anymore. It should also set the `DISPLAY` variable automatically.